tools/setup

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+ROOT=$(dirname $0)/..
+
+# 1. Install nvm
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | sh 
+
+# 1.1 ensure nvm is in your current process
+source ~/.nvm/nvm.sh
+
+# 2. Install node
+nvm install 10
+
+# 3. Update npm to later version
+npm install -g npm
+
+# 4. Install node_modules
+(cd $ROOT && npm install)
+
+
+# 5. Build the project
```

Comment:
maybe end with a list of suggestions instead of running sigh right away?

Reply:
Added a few suggestions. Let me know if you like them!